libs/partners/nvidia-ai-endpoints/langchain_nvidia_ai_endpoints/_common.py

```diff
@@ -439,10 +438,8 @@ class _NVIDIAClient(BaseModel):
     client: NVEModel = Field(NVEModel)
 
     model: str = Field(..., description="Name of the model to invoke")
-
-    temperature: float = Field(0.2, le=1.0, gt=0.0)
-    top_p: float = Field(0.7, le=1.0, ge=0.0)
-    max_tokens: int = Field(1024, le=1024, ge=32)
```

Comment:
will this be a breaking change? Can you still access like `llm = ChatNVIDIA(); llm.temperature`?

Would be good to handle access case at a minimum. Setting case is less important to me but could be nice for backwards compat!

Reply:
I have a slight preference for keeping these args top-level also if we're validating based on the list below as well. What do we gain by storing in a dict instead of as explicit pydantic properties?

Reply:
Can remove validation with something like `temperature: float = 0.2`

Reply:
An issue is that binding had no validation previously, which was the default way advertised in the docs.